app/javascript/packs/dashboard.ts

```diff
@@ -10,6 +10,8 @@ import type { Props as BackendsWidgetProps } from 'Dashboard/components/Backends
 const productsContainerId = 'products-widget'
 const backendsContainerId = 'backends-widget'
 
+import './dashboard.scss'
```

Comment:
WDYT about putting this import before both containers?

Reply:
Yes, this is wrong, `import/order` should've caught it. Thanks!